vllm/engine/async_llm_engine.py

```diff
@@ -417,7 +417,7 @@ async def generate(
             request.
         """
         # Preprocess the request.
-        arrival_time = time.time()
+        arrival_time = time.monotonic()
```

Comment:
Does this comment make sense? If so, could you add it?
```suggestion
        # This should not be used for logging, as it is monotonic time.
        arrival_time = time.monotonic()
```